File: app/src/main/java/com/digitex/designertools/ui/DesignerToolsActivity.java
package com.digitex.designertools.ui;

import android.app.Activity;
import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.TextView;

import com.digitex.designertools.R;
import com.digitex.designertools.utils.LaunchUtils;

public class DesignerToolsActivity extends Activity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_designer_tools);
        if (!LaunchUtils.isCyanogenMod(this)) {
            TextView tv = (TextView) findViewById(R.id.qs_tiles_section);
            tv.setText(R.string.overlays_section_text);
        }
        View headerGlyph = findViewById(R.id.header_glyph);
        if (headerGlyph != null) {
            headerGlyph.setOnClickListener(mGlyphClickListener);
        }
    }

    private View.OnClickListener mGlyphClickListener = new View.OnClickListener() {
        @Override
        public void onClick(View view) {
            startActivity(new Intent(DesignerToolsActivity.this, CreditsActivity.class));
        }
    };
}

File: app/src/main/java/com/digitex/designertools/ui/DesignerToolsActivity.kt
package com.digitex.designertools.ui

import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity
import androidx.core.view.updatePadding
import com.digitex.designertools.R
import com.digitex.designertools.ext.doOnApplyWindowInsets
import com.digitex.designertools.utils.isCyanogenMod
import kotlinx.android.synthetic.main.activity_designer_tools.*

class DesignerToolsActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_designer_tools)
        if (!isCyanogenMod()) {
            qsTilesTextView.setText(R.string.overlays_section_text)
        }

        rootView.systemUiVisibility = View.SYSTEM_UI_FLAG_LAYOUT_STABLE or
                View.SYSTEM_UI_FLAG_LAYOUT_HIDE_NAVIGATION

        contentView.doOnApplyWindowInsets { view, insets, padding ->
            view.updatePadding(
                    left = padding.left + insets.systemWindowInsetLeft,
                    top = padding.top + insets.systemWindowInsetTop,
                    right = padding.right + insets.systemWindowInsetRight,
                    bottom = padding.bottom + insets.systemWindowInsetBottom
            )
        }
    }
}
